libs/core/src/lib/combobox/combobox.component.ts

```diff
@@ -117,6 +120,12 @@ export class ComboboxComponent implements ControlValueAccessor, OnInit, OnChange
     @Input()
     itemTemplate: TemplateRef<any>;
 
+    /**
+     * The object attribute by which to group items.
+     */
+    @Input()
+    groupBy: string;
```

Comment:
I know we had converstaion about this idea, but maybe `groupFunction` wl be beter there. I just realised that there can be more complex/indent objects.